ui/src/api/aws.js

```diff
@@ -53,6 +53,15 @@ export function fetchPlates(experimentId, status) {
 }
 
 export function saveExperimentPlates(experimentName, status, plateMaps) {
+  return new Promise((resolve, reject) => {
+    let plateMapsToSave = lzutf8.compress(JSON.stringify(plateMaps), {
+      outputEncoding: 'Base64',
+    });
+      saveToDB(experimentName, status, 0, plateMapsToSave, reject, resolve);
+  });
+}
+
+export function publishExperimentPlates(experimentName, status, plateMaps) {
```

Comment:
Can you add some comments to each function to denote the difference? Maybe we should change the name of saveExperimentPlates to saveDraftPlates? Just a way to denote why they are different, as publish and save are to similar in my opinion

Reply:
Comments have now been added.